rpc/src/services/base_services.rs

```diff
@@ -23,7 +23,22 @@ use tezos_messages::ts_to_rfc3339;
 
 pub type BlockOperationsHashes = Vec<String>;
 
+use cached::proc_macro::cached;
+use cached::SizedCache;
+use cached::TimedCache;
+use cached::TimedSizedCache;
+
+pub const TIMED_SIZED_CACHE_SIZE: usize = 10;
+pub const TIMED_SIZED_CACHE_TTL_IN_SECS: u64 = 60;
+
 /// Retrieve blocks from database.
+#[cached(
```

Comment:
@tizoc @mambisi @jurajselep 
I am removing this from cache,
because, if the first call returns empty vec,
the next call will be cached for the empty, which is no good,
and also this is not even re-cached or warm-up-ed,


I dont know how this could be done by these annotations,
this will need to be cached (programatically) only if the result is ok and not empty vec,

I am collecting TODO stuff here